### PR TITLE
Settings, Keybinds, and Keybord-Nav UI Version 3

### LINKED ISSUE #55: FEAT settings page
**Is your feature request related to a problem? Please describe.**
Adding user preferences page.

### LINKED ISSUE #105: BUG-swipeoff-keyboard
**Describe the bug**
Swipe away keyboard on mobile.

**To Reproduce**
Click on a text field to make the keyboard appear. Try to get rid of keyboard. Get frustrated.

**Expected behavior**
Get rid of keyboard by swiping.

**Smartphone (please complete the following information):**
 - Device: Uh, all of them?

**Additional context**
[Discord source](https://discordapp.com/channels/691409213653778444/723796928739016704/737121969392189461)

### LINKED ISSUE #160: FEAT Local backup of online account data
Had a brief time when I couldn't login, so I couldn't access my data. Having the ability to at least have a local backup of my data would make me significantly more comfortable using condution, especially as the program/app is still new.

It would be nice if each of the desktop clients (Mac, Windows, Linux) stored a local copy of the task list *somewhere*. On Linux with the appimage this is obviously a bit clunky, but I think it's necessary regardless.

Preferably, the file would be saved in a format that is both computer and human readable, maybe just
```
projectName:task:description:start:end
```
of course, this is only an example, whatever the separation character it'd likely need to be something that is *invalid* for input in the actual gui, to make parsing easy.

### LINKED ISSUE #247: FEAT keyboard shortcuts
It'd be very useful if there were shortcuts to navigate around in the desktop client (I personally use Manjaro Linux / Gnome). It'd be great if the shortcut were describe as a hint.

For example, to create a project you could press "CTRL - P" and if you placed the mouse cursor over the sign "+" next to Projects a hint stating "CTRL - P" would appear

Thanks!

### LINKED ISSUE #264: BUG pressing Enter when FAB is highlighted on Project/Perspective pages switches to item title
**Describe the bug**
pressing Enter when FAB is highlighted on Project/Perspective pages switches to item title

**To Reproduce**
1. Go to a project/perspective
2. Highlight the FAB
3. Type something
4. Tap enter

**Expected behavior**
The task added to the inbox

**Smartphone (please complete the following information):**
 - Device: Android

**Additional context**
Email support request

### PR DESCRIPTION
**Pull request by Jemoka**
image

image

image

Settings page, keybind editor, `1.2.0` booya!

99.978% of the credit goes to @TheEnquirer as the engineering manager (and engineer and tester and developer) on this release.

Closes #55 
Closes #160 
Closes #247 
Closes #105 
Closes #264

### CODE REVIEW

**Comment by Jemoka** [comment]
@TheEnquirer are we ready to push into staging?

**Comment by Jemoka** [comment]
Can't merge. Missing new release notes. @FlyN-Nick want to have some fun typing up the notes?

**Comment by Jemoka** [comment]
actually we can prox push to staging and worry about that later.

**Review by TheEnquirer — DISMISSED** [review]
that's some nice code

File: justfile
Comment (zbuster05) [inline justfile:1]: @TheEnquirer I'm not sure this is necessary.

**Review by zbuster05 — DISMISSED** [review]
Other than the justfile, looks pretty good now.

**Comment by Jemoka** [comment]
I would hold off reviews until new release notes, etc. we will need to dismiss a few reviews for a bit

File: justfile
Comment (TheEnquirer) [reply justfile:1]: Anyone can run the project now without searching for the command. This should be a standard in all future projects. Certainly will be for mine.

**Comment by Jemoka** [comment]
Good for staging. Approve when ready.

**Review by TheEnquirer — DISMISSED** [review]
look at that good stuff :eyes:

**Review by TheEnquirer — DISMISSED** [review]
not so semi anymore are you.

File: justfile
Comment (zbuster05) [reply justfile:1]: Our specific use-case lacks any complex or even existent build recipe. Our build system wrapper does not require a wrapper.

**Comment by FlyN-Nick** [comment]
Thank you for making the fixes @TheEnquirer
[don't allow repeat keys inside the keybinds](https://github.com/Shabang-Systems/Condution/pull/266/commits/e0bf27c83eb0dd06348eef1ea84f6d76167f2bc8) and @TheEnquirer
[max keybind len w/ no overflow](https://github.com/Shabang-Systems/Condution/pull/266/commits/f9b03b5f3e3ea193f993ccc94bc727624099d604).

Love the polish and UI!

**Review by Jemoka — COMMENTED** [review]
what's the alternative to the command pallet command again now with removal? Sorry for the ping

**Review by Exr0n — DISMISSED** [review]
no changes?

**Review by TheEnquirer — APPROVED** [review]
hell yeah.

**Review by FlyN-Nick — APPROVED** [review]
Found no more bugs, looks good!

**Pull request merged**
